### PR TITLE
sdi-702 add test and improve missing data handling

### PR DESCRIPTION
**Pull request by lbennett-moj**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/sentencing/SentencingAdjustmentsToNomisIntTest.kt [hunk 1]
```diff
@@ -12,6 +12,7 @@ import org.awaitility.kotlin.await
 import org.awaitility.kotlin.matches
 import org.awaitility.kotlin.untilAsserted
 import org.awaitility.kotlin.untilCallTo
+import org.joda.time.LocalDate
```
Comment (steverendell) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/sentencing/SentencingAdjustmentsToNomisIntTest.kt:15]: joda?

Comment (lbennett-moj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/sentencing/SentencingAdjustmentsToNomisIntTest.kt:15]: whoops

Comment (lbennett-moj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/sentencing/SentencingAdjustmentsToNomisIntTest.kt]: just corrected

**Review by steverendell — APPROVED** [review]
lgtm

**Pull request merged**